src/js/pages/jobs/JobDetailPage.js

```diff
@@ -143,21 +149,34 @@ class JobDetailPage extends mixin(StoreMixin, TabsMixin) {
   }
 
   getActionButtons() {
-    let dropdownItems = [
-      {
-        className: 'hidden',
-        html: 'More',
-        id: JobActionItem.MORE
-      },
-      {
+    let job = MetronomeStore.getJob(this.props.params.id);
+    let dropdownItems = [];
+    let [schedule] = job.getSchedules();
+
+    dropdownItems.push({
+      className: 'hidden',
+      html: 'More',
+      id: JobActionItem.MORE
+    });
+
+    if (schedule != null && schedule.enabled) {
```

Comment:
In this specific case I guess there won't be a significant performance difference. This is not the point.
It's about approaching solutions and logical simplicity. Your approach is more complex on the logical level.

The Apple I by Steve Wozniak for example was a masterpiece of logical connections, thus it needed less chips (expensive).

Another example which points to a similar direction:
If I see 4/6, I will shorten it to 2/3, for the sake of simplicity.

So, I really don't care if you change it or not. Just wanted to clarify my point